### ISSUE TITLE
Add RouteModel to modelSourcePool

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#89]
_From @n8rzz on November 26, 2016 23:9_

_Copied from original issue: n8rzz/atc#166_

**Comment by n8rzz** [comment]
@erikquinn is the feasible with the updates to `RouteModel`?  No need to tackle this now as part of #24 or #899, just want to call it out so it's on your radar.

**Comment by erikquinn** [comment]
Absolutely. Way I see it, anything that's frequently `new`ed should be in there, and routes would _definitely_ be one of those things.
